benchmarks/src/bin/tpch.rs

```diff
@@ -75,10 +75,9 @@ struct BallistaBenchmarkOpt {
     #[structopt(short = "i", long = "iterations", default_value = "3")]
     iterations: usize,
 
-    /// Batch size when reading CSV or Parquet files
-    #[structopt(short = "s", long = "batch-size", default_value = "8192")]
-    batch_size: usize,
-
+    // /// Batch size when reading CSV or Parquet files
```

Comment:
These parameters are never read (aka they don't do anything) so removing them from the CLI seemed like a reasonable thing to do